.env.example

```
@@ -4,6 +4,9 @@ HOST="0.0.0.0"
 UVICORN_RELOAD=true
 UVICORN_LOG_LEVEL="info"
 
+# DevCycle Feature Flags Configuration
+DEVCYCLE_SERVER_SDK_KEY="your_devcyle_server_sdk_key_here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)